docs/api/jax.rst

```diff
@@ -4,10 +4,10 @@
     See LICENSE for license information.
 
 Jax
-=======
+===
 
 Pre-defined Variable of Logical Axes
-------------------------------------
+-------------------------------------
```

Comment:
Was this needed to fix the rendering? The earlier version seems fine and of the correct length